app/Repositories/SubuserRepository.php

```diff
@@ -170,47 +164,31 @@ public function create($sid, array $data)
                         array_push($daemonPermissions, $this->permissions[$permission]);
                     }
 
-                    $model = new Models\Permission;
-                    $model->fill([
-                        'user_id' => $user->id,
-                        'server_id' => $server->id,
+                    Models\Permission::create([
+                        'subuser_id' => $subuser->id,
                         'permission' => $permission,
                     ]);
-                    $model->save();
                 }
             }
 
             // Contact Daemon
             // We contact even if they don't have any daemon permissions to overwrite
             // if they did have them previously.
 
-            $node = Models\Node::getByID($server->node);
-            $client = Models\Node::guzzleRequest($server->node);
-
-            $res = $client->request('PATCH', '/server', [
-                'headers' => [
-                    'X-Access-Server' => $server->uuid,
-                    'X-Access-Token' => $node->daemonSecret,
-                ],
+            $server->node->guzzleClient([
```

Comment:
I believe this was because it depends entirely on the user making the call, which isn't necessarily available to the node.

I can look at that again though.